src/Ztm.Data.Entity.Testing/TestMainDatabase.cs

```diff
@@ -19,6 +19,52 @@ protected override void ConfigureWebApiCallback(ModelBuilder modelBuilder)
                 b.Property(e => e.RequestIp).HasConversion(Converters.IPAddressToStringConverter);
                 b.Property(e => e.Url).HasConversion(Converters.UriToStringConverter);
             });
+
+            modelBuilder.Entity<Block>(b =>
+            {
+                b.Property(e => e.Hash).HasConversion(Converters.UInt256ToBytesConverter);
+                b.Property(e => e.MerkleRoot).HasConversion(Converters.UInt256ToBytesConverter);
+                b.Property(e => e.MtpHashValue).HasConversion(Converters.UInt256ToBytesConverter);
+                b.Property(e => e.Reserved1).HasConversion(Converters.UInt256ToBytesConverter);
+                b.Property(e => e.Reserved2).HasConversion(Converters.UInt256ToBytesConverter);
+            });
+
+            modelBuilder.Entity<BlockTransaction>(b =>
+            {
+                b.Property(e => e.BlockHash).HasConversion(Converters.UInt256ToBytesConverter);
+                b.Property(e => e.TransactionHash).HasConversion(Converters.UInt256ToBytesConverter);
+            });
+
+            modelBuilder.Entity<Input>(b =>
+            {
+                b.Property(e => e.TransactionHash).IsRequired().HasConversion(Converters.UInt256ToBytesConverter);
+                b.Property(e => e.OutputHash).IsRequired().HasConversion(Converters.UInt256ToBytesConverter);
+            });
+
+            modelBuilder.Entity<Output>(b =>
+            {
+                b.Property(e => e.TransactionHash).IsRequired().HasConversion(Converters.UInt256ToBytesConverter);
+            });
+
+            modelBuilder.Entity<Transaction>(b =>
+            {
+                b.Property(e => e.Hash).IsRequired().HasConversion(Converters.UInt256ToBytesConverter);
+            });
+
+            modelBuilder.Entity<WatchingBlock>(b =>
+            {
+                b.Property(e => e.Hash).IsRequired().HasConversion(Converters.UInt256ToBytesConverter);
+            });
+
+            modelBuilder.Entity<WatchingTransaction>(b =>
+            {
+                b.Property(e => e.Hash).IsRequired().HasConversion(Converters.UInt256ToBytesConverter);
+            });
+
+            modelBuilder.Entity<WebApiCallback>(b =>
+            {
+                b.Property(e => e.TransactionId).HasConversion(Converters.UInt256ToBytesConverter);
+            });
```

Comment:
If we want to put all of these logic in a single method we should override `OnModelCreating` instead of `ConfigureWebApiCallback`.